src/pages/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Home from './pageComponents/Home'
+
+export default () => {
+   return(
```

Comment:
Nit: Most JavaScript style guides recommend putting a space between the `return` keyword and an opening parenthesis.

Also, as a more general comment, we should probably make sure that our linters and code styles are consistent between this project and askdarcel-web, so that people who work on both of them don't have to remember different sets of style rules.

Reply:
Thanks! I really had to search for what "nit:" is. :)